### ISSUE TITLE
Not supported OFFSET

### ISSUE THREAD

**Issue by PumpkinSeed** [xwb1989/sqlparser#12]
I got an error for a query which contain OFFSET. It should be a valid query. 
```
syntax error at position 42 near OFFSET for query - SELECT * FROM AccessToken LIMIT 10 OFFSET 13
```

**Comment by bramp** [comment]
This issue seems fixed by d093d029e2170e41f99261ab9b8e947cf602a1d6 (Imported the latest https://github.com/youtube/vitess sqlparser)

**Comment by bramp** [comment]
Closing this issue. Feel free to reopen with an example if you still encounter the issue.
